### PR TITLE
[DDW-617][DDW-626] Update Catalyst voting registration text copy and fix registration issues

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR updates Catalyst voting registration text copy for Fund4 and fixes the issue of voting registration being stuck in pending/confirming states.

Text copy documents:

- https://docs.google.com/document/d/1dowB1Cc-d-030RoiAOByDDXKICc9S_-81U-x0V_zTGA/edit
- https://docs.google.com/document/d/1QVzT3VtseXPFhv5Q5DTihjKX46TuzkCT7H6MrcUYTjE/edit#

Important dates:

- Fund-4 Registration start: 06/Apr/2021, 19:00 PM UTC
- Fund-4 Registration end: 13/Apr/2021, 19:00 PM UTC
- Fund-4 Snapshot start: 13/Apr/2021, 19:00 PM UTC
- Fund-4 Voting start: 15/Apr/2021, 19:00 PM UTC
- Fund-4 Voting end: 05/May/2021, 19:00 PM UTC
- Fund-5 Registration start: 18/May/2021, 19:00 PM UTC
- Fund-5 Registration end: 25/May/2021, 19:00 PM UTC
- Fund-5 Snapshot start: 25/May/2021, 19:00 PM UTC

### Important
**This PR also increases the maximum event listeners limit to avoid IPC channel stalling as well as improves and speeds up the TLS/HTTPS connections setup by adding TLS session caching and socket re-using.**

You can toggle connection socket stats logging by opening Dev Console and entering `logSocketStats = true;` or `logSocketStats = false;`

## Todos

- [x] Disable voting on Testnet

## Screenshots

Screenshot 2021-03-27 at 12 19 57
Screenshot 2021-03-27 at 12 20 17

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test voting registration and make sure text copy looks good

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by nikolaglumac** [comment]
I have added the "DO NOT MERGE" label so that we don't forget to disable voting registration for Testnet before merging.

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested on Testnet 17362. Great work @nikolaglumac 👍

**Pull request merged**
